test/e2e/conditions.go

```diff
@@ -29,6 +29,10 @@ func conditionsToYaml(conditions nmstatev1alpha1.ConditionList) string {
 	return string(manifest)
 }
 
+func enactmentKey(node string, policy string) types.NamespacedName {
```

Comment:
This can cause collisions in case someone is trying to hard to break us. Can we get a different separator? Maybe the escaped slash used in resource names?

Reply:
You mean something like this ?

https://github.com/intel/multus-cni/blob/master/examples/sample-pod.yml#L7

Reply:
Yes, but I'm not sure it is allowed in object names. I think slash would be way better for NNCE than dash.

Reply:
humm not working
```
"node01/test-policy\" is invalid: metadata.name: Invalid value: \"node01/test-policy\":
```